catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/ContextualSearch.java

```diff
@@ -11,27 +11,32 @@
  * is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package ddf.catalog.source.opensearch;
+package ddf.catalog.source.opensearch.impl;
+
+import java.util.Map;
 
 public class ContextualSearch {
-    private String searchPhrase;
+
+    // A map of search phrases to be used to build the OpenSearch URL parameters
+    // The default search phrase is "q"
+    private Map<String, String> searchPhraseMap;
 
     private String selectors;
 
     private boolean isCaseSensitive;
 
-    public ContextualSearch(String selectors, String searchPhrase, boolean isCaseSensitive) {
+    public ContextualSearch(String selectors, Map<String, String> searchPhraseMap, boolean isCaseSensitive) {
```

Comment:
The comment here https://github.com/codice/ddf/pull/2182/files#diff-9c053948d9fb1c8b27085efd3ab184feR20 describes what the map is used for, but there wasn't a comment added for the `why`.  This was changed to a map so other textual searches can be added as parameters to the OpenSearch query (other than just being restricted to `q`).